bridger/utils/options1.json

```diff
@@ -370,7 +370,7 @@
         "float_field",
         "decimal_field"
     ],
-    "filter_fields": {
+    "filterset_fields": {
```

Comment:
This change also means, that you have to adjust this in all modules, right?

Reply:
There is no reference to `filter_fields` in wb repo, I will check the other

Reply:
No reference to filter_fields in any other modules